content/execution_model.tex

```diff
@@ -45,7 +45,7 @@ \subsection{Progress of \openshmem Operations}\label{subsec:progress}
       hardware offload capabilities and provide asynchronous one-sided
       communication without software assistance.
   \item Implementations should avoid deferring the execution of one-sided
-      operations until a synchronization point where data is known to be
+      operations until a synchronization point, where data is known to be
```

Comment:
See _Comma Abuse No. 15_ in [Purdue's Extended Rules for Using Commas](https://owl.english.purdue.edu/owl/resource/607/02/).